app/screens/send-bitcoin-screen/send-bitcoin-confirmation-screen.tsx

```diff
@@ -560,4 +567,7 @@ const useStyles = makeStyles(({ colors }) => ({
     justifyContent: "space-between",
     alignItems: "center",
   },
+  sliderContainer: {
+    padding: 20,
```

Comment:
should we make it even a bit bigger?

```suggestion
    padding: 32,
```

Reply:
merging for now. we can change in a future PR later.

Reply:
Yes, this may help.